internal/admission/validation/consumers/credentials/vars.go

```diff
@@ -17,6 +17,19 @@ var SupportedTypes = sets.NewString(
 	"mtls-auth",
 )
 
+var ValidTypes = sets.NewString(
```

Comment:
We could add a comment and make sure that `SupportedTypes` is always included in the `ValidTypes`.
```suggestion
// ValidTypes are all types considered as valid. It's a sum of `SupportedTypes` (the ones that KIC reconciles) and other credential types supported outside of KIC (i.e. in KGO).
var ValidTypes = sets.NewString(append(
	SupportedTypes.UnsortedList(),
	// Secrets with `konghq.com/cerdential=konnect is used by KGO.
	// KIC should not reconcile them, but should accept them in validation webhook.
	"konnect",
)...)
```

Reply:
Done.